### ISSUE TITLE
Docker

### ISSUE THREAD

**Issue by jimtng** [spinza/tuya_mqtt#2]
Great project! I'd like to make a request for a docker version please :)

**Comment by spinza** [comment]
I haven't built docker containers before.  Have messed around with them. 

If the `devices.json` file is available already I think it should be simple, but getting that file is a bit of a mission (per the README).  How would you suggest that's done?

**Comment by jimtng** [comment]
All configurations are to be mounted in an external volume that is editable outside docker.

**Comment by spinza** [comment]
Yes, so if you have the config it would work, but the `devices.json` needs to be generated with the packages installed and it's kind of manual.  Any suggestions on how to dockerize this somewhat manual part? The process is described in this project's README and the `tinytua` documentation.
